routers/api/v1/repo.go

```diff
@@ -191,6 +191,28 @@ func CreateOrgRepo(ctx *middleware.Context, opt api.CreateRepoOption) {
 	createRepo(ctx, org, opt)
 }
 
+func ForkRepo(ctx *middleware.Context) {
```

Comment:
This is missing a request form which now can't fork a repository to organization.